src/components/EditorStageBar.vue

```diff
@@ -34,15 +29,18 @@ export default {
       current: this.initialStage,
       stages: [
         {
-          label: '<span>1. Enter Template <span class="bolder">Settings</span></span>',
+          label:
+            '<span>1. Enter Template <span class="bolder">Settings</span></span>',
           name: EDITOR_STAGES.SETTINGS,
         },
         {
-          label: '<span class="link">2. Select your <span class="bolder">Data Sources<span></span>',
+          label:
+            '<span class="link">2. Select your <span class="bolder">Data Sources<span></span>',
```

Comment:
The color of the stage name is purple and it should be white
<img width="666" alt="Captura de pantalla 2020-05-25 a las 17 24 06" src="https://user-images.githubusercontent.com/16032491/82825992-8f106000-9eac-11ea-932d-92f04b4b0737.png">